doc/classes/Color.xml

```diff
@@ -362,8 +362,8 @@
 		<constant name="coral" value="Color( 1, 0.5, 0.31, 1 )">
 			Coral color.
 		</constant>
-		<constant name="cornflower" value="Color( 0.39, 0.58, 0.93, 1 )">
-			Cornflower color.
+		<constant name="cornflowerblue" value="Color( 0.39, 0.58, 0.93, 1 )">
+			Cornflower Blue color.
```

Comment:
Should be "Cornflower blue".

Reply:
Right, keeping with the rest of the comments.